tests/comfy_cli/test_file_utils.py

```
@@ -81,3 +84,86 @@ def test_zip_files_without_git_falls_back_to_walk(tmp_path, monkeypatch):
+
+
+def test_atomic_write_text_overwrites_existing(tmp_path):
+    target = tmp_path / "out.txt"
+    target.write_text("old", encoding="utf-8")
+
+    atomic_write_text(target, "new")
+
+    assert target.read_text(encoding="utf-8") == "new"
+
+
+def test_atomic_write_text_fsync_true_still_writes(tmp_path):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)